server/github.go

```diff
@@ -167,6 +167,29 @@ func (s *Server) getComments(ctx context.Context, pr *model.PullRequest) (commen
 	return allComments, nil
 }
 
+func (s *Server) getFiles(ctx context.Context, repoOwner, repoName string, number int) ([]*github.CommitFile, error) {
```

Comment:
This is actually for a PR. Do you mean prNumber? It is exactly because PRs and issues are treated the same in the API, that's why I used `number`. It's the same in `getComments` too.

Reply:
Yeah, from github side PRs and issues have an "issue number", so we can use `issueNumber` for PRs too. In fact, we can update every occurrence of `prNumber` or `number` with `issueNumber`.

Reply:
Ah I see. Sure.